plover/dictionary/test_loading_manager.py

```diff
@@ -25,8 +25,6 @@ def __call__(self, filename):
         loader = MockLoader(files)
         with patch('plover.dictionary.loading_manager.load_dictionary', loader):
             manager = loading_manager.DictionaryLoadingManager()
-            manager.start_loading('a')
-            manager.start_loading('b')
```

Comment:
This line tested that we only loaded file 'b' once even though we tried to load it twice.